web/src/components/Article/components/FullLayout/FullLayout.tsx

```diff
@@ -119,8 +147,30 @@ const FullLayout = ({ article }: Props) => {
                   className="pb-1 text-slate-500"
                 />
               </div>
-              <PostThumbsCell postId={article.id} />
+              {article.id && <PostThumbsCell postId={article.id} />}
```

Comment:
Nice fix, this was giving those 400 errors in the create article preview right?

Reply:
Yes it was!